lib/data/getOdMatrix.ts

```diff
@@ -18,19 +18,19 @@ type od = {
   props: GeoJsonProperties;
 };
 
-const getOdMatrix = async () => {
-  const flows = await getODFlows();
+const getOdMatrix = async (...args: string[]) => {
+  const flows = await getODFlows(args[0]);
```

Comment:
It is not clear to me why we use an array of strings if we only ever use its first element.

```suggestion
const getOdMatrix = async (departmentId: string) => {
  const flows = await getODFlows(departmentId);
```